src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -286,6 +286,21 @@ public function initialize_nodes_to_translate() {
 					),
 				),
 			),
+			'blockquote'       => array(
```

Comment:
I've changed the coding style settings in Scrutinizer for this project: next tests should not throw these errors, as we use tabs instead of spaces.